src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+    },
+
+    subtract(value) {
+      this.result -= value;
+    },
+
+    multiply(value) {
+      this.result *= value;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        this.result /= value;
+      }
+    },
```

Comment:
While it's good to check for division by zero, consider throwing an error or handling the zero division case differently. Silently failing (not doing the division) may lead to unexpected behavior if not properly documented.